extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -65,9 +84,25 @@ export class AmpSelector extends AMP.BaseElement {
       this.element.setAttribute('aria-disabled', 'true');
     }
 
+    const kbSupportMode = this.element.getAttribute('keyboard-support');
+    if (kbSupportMode == 'none') {
+      this.kbSupportMode_ = KB_SUPPORT.NONE;
+    } else if (kbSupportMode == 'select') {
+      this.kbSupportMode_ = KB_SUPPORT.SELECT;
+      user().assert(!this.isMultiple_,
+        '[keyboard-support=select] not supported for ' +
+        'multiple selection amp-selector');
+    } else {
+      this.kbSupportMode_ = KB_SUPPORT.FOCUS;
```

Comment:
I have been thinking about this for bit. As much as I want the default to be 'focus', I am worried that unless amp-select is used to create visually-obvious grouping of options (like tabs, custom select list, etc..) change of behaviour may be unexpected. So let's default to `None`